packages/react-primitives/src/components/dropdown/dropdown.css.ts

```diff
@@ -38,6 +37,9 @@ export const label = style({
   fontSize: theme.font.size.xs,
   fontWeight: theme.font.weight.semibold,
   color: publicVariables.input.label.color,
+  position: "absolute",
```

Comment:
Can we avoid this? Are we sure it's going to look ok in all circumstances?

Reply:
Not sure, part of the fix was moving the `<label>` outside of the `<Select.Root>`, but with this UI it's inside of the Select box